tests/Fixer/Phpdoc/NoBlankLinesAfterPhpdocFixerTest.php

```diff
@@ -160,6 +160,20 @@ public function testLineBeforeUseStatementIsNotRemoved(): void
         $this->doTest($expected);
     }
 
+    public function testLineBeforeRequireIsNotBeRemoved(): void
+    {
+        $expected = <<<'EOF'
+<?php
+/**
+ * This describes what my script does.
+ */
+
+require 'vendor/autoload.php';
```

Comment:
Since 4 new tokens were added to the fixer we should also have test for each. It can be done  within the same test method by yielding 4 cases in a loop.